components/store/StoreCard.js

```diff
@@ -55,89 +61,153 @@ export default function StoreCard({
+                    seeProduct
+                  ),
+                }}
+                numberOfLines={1}
+                ellipsizeMode="tail">
+                {storeName}
+              </Subhead>
+            ) : (
+              <Title
+                style={{
+                  maxWidth: getMaxWidth(
+                    Dimensions.get('window').width,
+                    snapOrEbtAccepted,
+                    seeProduct
+                  ),
+                }}
+                numberOfLines={1}
+                ellipsizeMode="tail">
+                {storeName}
+              </Title>
+            )}
+          </RowContainer>
+          {seeProduct && <StoreProductButton callBack={callBack} />}
+        </SpaceBetweenRowContainer>
+        {/* Accepted Program Tags */}
+        <InLineContainer
+          style={{ flexWrap: 'wrap', paddingTop: 6, paddingBottom: 6 }}>
           {snapOrEbtAccepted && (
-            <EBTStatusBar>
-              <FontAwesome5 name="check" size={10} color={Colors.darkerGreen} />
-              <Caption color={Colors.darkerGreen}> EBT</Caption>
-            </EBTStatusBar>
+            <Chip
+              icon={() => (
+                <FontAwesome5
+                  name="credit-card"
+                  size={10}
+                  color={Colors.darkerGreen}
+                  style={{ marginTop: -10 }}
+                />
+              )}
+              textStyle={styles.chipText}
+              style={styles.chip}>
+              <Caption color={Colors.darkerGreen}>EBT</Caption>
+            </Chip>
           )}
-        </RowContainer>
-        {seeProduct && <StoreProductButton callBack={callBack} />}
-      </SpaceBetweenRowContainer>
-      {seeDistance && (
-        <Caption style={{ marginBottom: 4 }} color={Colors.secondaryText}>
-          {`${distance} miles away`}
-        </Caption>
-      )}
-
-      <InLineContainer style={{ alignItems: 'center' }}>
-        <FontAwesome5
-          name="star"
-          solid
-          size={16}
-          color={rewardsAccepted ? Colors.primaryGreen : Colors.secondaryText}
-        />
-        <StoreDetailText greenText={rewardsAccepted}>
-          {rewardsAccepted
-            ? 'Earn and redeem Healthy Rewards here'
-            : 'Healthy Rewards not accepted'}
-        </StoreDetailText>
-      </InLineContainer>
-      <TouchableOpacity
-        onPress={openDirections}
-        onLongPress={writeAddressToClipboard}>
+          {wic && (
+            <Chip
+              icon={() => (
+                <FontAwesome5
+                  name="heart"
+                  solid
+                  size={10}
+                  color={Colors.darkerGreen}
+                  style={{ marginTop: -10 }}
+                />
+              )}
+              textStyle={styles.chipText}
+              style={styles.chip}>
+              <Caption color={Colors.darkerGreen}>WIC</Caption>
+            </Chip>
+          )}
```

Comment:
Can probably be a separate component since only the caption and font name is changed for similar components below!

Reply:
Realised @wangannie also noted this in her review